src/s1reader/s1_reader.py

```diff
@@ -686,6 +691,16 @@ def burst_from_xml(annotation_path: str, orbit_path: str, tiff_path: str,
                                                                          aux_cal_subswath,
                                                                          sensing_start)
 
+        # Extended FM and DC coefficient information
+        if mitigate_az_fmrate_mismatch:
+            extended_coeffs = \
+                BurstExtendedCoeffs.from_polynomial_lists(az_fm_rate_list,
+                                                        doppler_list,
+                                                        sensing_start,
+                                                        sensing_start + sensing_duration)
+        else:
+            extended_coeffs = None
```

Comment:
```suggestion
        # Extended FM and DC coefficient information
        extended_coeffs = BurstExtendedCoeffs.from_polynomial_lists(az_fm_rate_list,
                                                                    doppler_list,
                                                                    sensing_start,
                                                                    sensing_start + sensing_duration)
```

See comment at `load bursts` about running this function every time, no flag needed